src/App.jsx

```diff
@@ -29,6 +30,7 @@ const App = () => {
           <Route path='/categories' element={<CategoryPage />} />
           <Route path='/login' element={<LoginPage />} />
           <Route path='/signup' element={<SingupPage />} />
+          <Route path='/seller' element={<SellerPage />} />
```

Comment:
I believe this route is for adding a new product by the seller, then it should be something like /seller/add-product